packages/xen-gnt/xen-gnt.2.2.3/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "john.else@citrix.com"
+authors: ["Anil Madhavapeddy" "John Else" "Thomas Leonard" "Andrew Cooper" "David Scott"]
+homepage:      "https://github.com/mirage/ocaml-gnt"
+bug-reports:   "https://github.com/mirage/ocaml-gnt/issues"
+dev-repo:      "https://github.com/mirage/ocaml-gnt.git"
+build: [
+  [make]
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"]
+]
+remove: [
+  [make "PREFIX=%{prefix}%" "uninstall"]
+]
+depends: [
+  "ounit"
```

Comment:
same as above... and maybe misses `ocamlfind` and `ocamlbuild` build time dependencies